### PR TITLE
blog: metrics collection

### PR DESCRIPTION
**Pull request by ydayagi**
https://issues.redhat.com/browse/ECOPROJECT-697
Signed-off-by: Yaron Dayagi <ydayagi@redhat.com>

### CODE REVIEW

File: _posts/2022-04-11-writing-metrics-to-control-plane.markdown [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - flotta
+  
+summary-1: Writing metrics to control plane
+---
+
+Flotta edge devices collect metrics from device's system and workloads. These metrics are stored locally in the device. How do we get the metrics to the control plane? How can we view the metrics in control plane? these questions are what this blog post is about.
+
+### Device metrics - quick overview
+
+Configuring a device to collect metrics is covered by the [device-metrics user guide](https://github.com/project-flotta/flotta-operator/blob/main/docs/user-guide/device-metrics.md). Flotta agent in the device scrapes system and workloads metrics and writes to a local TSDB folder. For system metrics we use `node_exporter`. You can find metrics such as `node_memory_MemAvailable_bytes`. The local TSDB enforces a retention policy.
```
[the remark this answers is not in the document]
Comment (ydayagi) [reply _posts/2022-04-11-writing-metrics-to-control-plane.markdown:18]: I don't think this is good English in that context. No one is supposed to thank anyone. But...I will change

[the remark this answers is not in the document]
Comment (jakub-dzon) [reply _posts/2022-04-11-writing-metrics-to-control-plane.markdown:18]: It's a figure of speech, not literal thanking anyone. If you don't feel comfortable with "Thanks to", use "Due to that" or "As a result of that".

File: _posts/2022-04-11-writing-metrics-to-control-plane.markdown [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "Writing metrics to control plane"
```
[the remark this answers is not in the document]
Comment (ydayagi) [reply _posts/2022-04-11-writing-metrics-to-control-plane.markdown:3]: @pkliczewski I thought that this is what the post is supposed to achieve.

File: _posts/2022-04-11-writing-metrics-to-control-plane.markdown [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+categories: flotta
+author: Yaron Dayagi
+tags:
+  - guide
+  - flotta
+  
+summary-1: Writing metrics to control plane
+---
+
+Flotta edge devices collect metrics from device's system and workloads. These metrics are stored locally in the device. How do we get the metrics to the control plane? How can we view the metrics in control plane? these questions are what this blog post is about.
```
[the remark this answers is not in the document]
Comment (ydayagi) [reply _posts/2022-04-11-writing-metrics-to-control-plane.markdown]: done

**Review by jakub-dzon — APPROVED** [review]

**Review by machacekondra — APPROVED** [review]

**Pull request merged**